### PR TITLE
ENYO-2313-Place holder is not shown while images are loading in DataG…

### PR DESCRIPTION
**Pull request by rajyavardhanp**
…ridListSample

Issue: Placeholder is the background of image (in this case) And image
doesn't have predefined width and height.
If the image doesn't load there is no way placeholder can have bounds.
Fix:  In layout/DatagridListImageItem, a class that has width n height
100% is added on 'image' control on condition (this.sizing) based.
Enyo-DCO-1.1-Signed-off-by:  Rajyavardhan P <rajyavardhan.p@lge.com>

### CODE REVIEW

File: lib/GridListImageItem/GridListImageItem.css [hunk 1]
```diff
@@ -14,6 +14,10 @@
 	height: 100%;
 	max-height: 100%;
 }
+.enyo-gridlist-imageitem .enyo-gridlist-imageitem-image {
```
Comment (aarontam) [inline lib/GridListImageItem/GridListImageItem.css]: This is fine, functionally, but it is a bit odd semantically that we only add this class under the specific case where we are not using a background-image and have specified a placeholder image; the name of the class itself seems too generic in this case.

File: lib/GridListImageItem/GridListImageItem.js [hunk 2]
```diff
@@ -191,6 +191,7 @@ module.exports = kind(
 	* @private
 	*/
 	imageSizingChanged: function () {
+		this.$.image.addRemoveClass('enyo-gridlist-imageitem-image', (!this.imageSizing && !!this.placeholder));
```
Comment (aarontam) [inline lib/GridListImageItem/GridListImageItem.js]: When we are using `imageSizing`, the `sized-image` class effectively adds the 100% width and height declarations on the image, so I am not sure why we need to make this conditional on *not* having `imageSizing` specified.

**Comment by aarontam** [comment]
Closing in favor of https://github.com/enyojs/layout/pull/140.

File: lib/GridListImageItem/GridListImageItem.js (hunk 2 above)
Comment (rajyavardhanp) [inline lib/GridListImageItem/GridListImageItem.js]: Hi @aarontam I didn't see 'sized-image' class was being applied here at that time. I thought it's 'enyo.Image' related class. So i wanted the affect to be minimal and confined.
One query: !!this.placeholder is also not required? If there is no placeholder letting the image to take a natural size ?

Comment (aarontam) [inline lib/GridListImageItem/GridListImageItem.js]: Hi @rajyavardhanp, if you take a look here, you will see that we were not intending to allow the image to have a natural size: https://github.com/enyojs/layout/blob/2.6.0-dev/src/GridListImageItem/GridListImageItem.css#L13.

**Pull request closed without merging**